File: CS/TemplateCardCaptionLocalization/MyLayoutView.cs
using DevExpress.XtraGrid.Views.Layout;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Layout.Customization;
using DevExpress.Skins;
using System.Windows.Forms;
using System.Reflection;
using System;

namespace TemplateCardCaptionLocalization {
    public class MyLayoutView :LayoutView {
        public MyLayoutView() : base() { }
        public MyLayoutView(GridControl grid) : base(grid) { }

        internal const string MyLayoutViewName = "MyLayoutView";
        protected override string ViewName {
            get { return MyLayoutViewName; }
        }

        protected override LayoutViewCustomizationForm CreateCustomizationForm() {
            SkinManager.EnableFormSkins();
            LayoutViewCustomizationForm form = new MyLayoutViewCustomizationForm(this);
            form.FormClosing += OnCustomizationFormClosing;
            return form;
        }

        void OnCustomizationFormClosing(object sender, FormClosingEventArgs e) {
            typeof(LayoutView).GetMethod("OnCustomizationFormClosing", BindingFlags.Instance |
                BindingFlags.NonPublic, null, new Type[] { typeof(object), typeof(FormClosingEventArgs) },
                new ParameterModifier[0]).Invoke(this, new object[] { sender, e });
        }
    }
}

File: VB/TemplateCardCaptionLocalization/MyLayoutView.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraGrid.Views.Layout
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Layout.Customization
Imports DevExpress.Skins
Imports System.Windows.Forms
Imports System.Reflection
Imports System

Namespace TemplateCardCaptionLocalization
	Public Class MyLayoutView
		Inherits LayoutView
		Public Sub New()
			MyBase.New()
		End Sub
		Public Sub New(ByVal grid As GridControl)
			MyBase.New(grid)
		End Sub

		Friend Const MyLayoutViewName As String = "MyLayoutView"
		Protected Overrides ReadOnly Property ViewName() As String
			Get
				Return MyLayoutViewName
			End Get
		End Property

		Protected Overrides Function CreateCustomizationForm() As LayoutViewCustomizationForm
			SkinManager.EnableFormSkins()
			Dim form As LayoutViewCustomizationForm = New MyLayoutViewCustomizationForm(Me)
			AddHandler form.FormClosing, AddressOf OnCustomizationFormClosing
			Return form
		End Function

		Private Sub OnCustomizationFormClosing(ByVal sender As Object, ByVal e As FormClosingEventArgs)
			GetType(LayoutView).GetMethod("OnCustomizationFormClosing", BindingFlags.Instance Or BindingFlags.NonPublic, Nothing, New Type() { GetType(Object), GetType(FormClosingEventArgs) }, New ParameterModifier(){}).Invoke(Me, New Object() { sender, e })
		End Sub
	End Class
End Namespace